include/sdbus-c++/Types.h

```diff
@@ -280,4 +287,14 @@ namespace sdbus {
 
 }
 
+template <size_t _I, typename... _ValueTypes>
+struct std::tuple_element<_I, sdbus::Struct<_ValueTypes...>>
+        : std::tuple_element<_I, std::tuple<_ValueTypes...>>
+{};
```

Comment:
This is an amazing neat solution. I would have never come up with this solution. Amazing :)

Reply:
Yes, neat, nice, simple (and standard-conforming) solution to make any class a tuple-like class :)